addon/adapters/rest.js

```diff
@@ -963,7 +963,7 @@ const RESTAdapter = Adapter.extend(BuildURLMixin, {
     if (this.isSuccess(status, headers, payload)) {
       return payload;
     } else if (this.isInvalid(status, headers, payload)) {
-      return new InvalidError(payload.errors);
+      return new InvalidError([payload.errors]);
```

Comment:
I believe this actually must be run through `this.normalizeErrorResponse`  -> https://github.com/emberjs/data/blob/2fabe2e86dfa6561d9d0038d4726e5fcc4658320/addon/adapters/rest.js#L1151-L1171

Like:

```js
new InvalidError(this.normalizeErrorResponse(status, headers, payload);
```